charts/rancher-turtles/values.yaml

```diff
@@ -3,3 +3,5 @@ rancherTurtles:
   tag: v0.0.0
   imagePullPolicy: Never
   namespace: rancher-turtles-system
+capiOperator:
+  enabled: true
```

Comment:
I would prefer leaving it on disabled by default for the out-of-cluster support

Reply:
I think its ok to have it enabled by default. The main is that there is an option to control it.

Reply:
you can do it, `--set cluster-api-operator.infrastructure=aws` should work. We tried it with cert-manager in CAPI operator

Reply:
Could you please update e2e tests so they won't break because of this change and incorporate the described functionality?